anda/rust/rustup/rustup.spec

```diff
@@ -0,0 +1,62 @@
+Name:           rustup
+Version:        1.25.1
+Release:        %autorelease
+URL:            https://rust-lang.github.io/rustup/
+License:        MIT, Apache-2.0
+Summary:        The Rust toolchain installer
+Source0:        https://github.com/rust-lang/rustup/archive/refs/tags/%{version}.tar.gz
+BuildRequires:  cargo pkgconfig(openssl)
```

Comment:
i would check fedora's [rust packaging guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Rust/) here. `rust-packaging` should be the build requirement